runtime/src/main/java/org/corfudb/runtime/view/AbstractView.java

```diff
@@ -83,6 +84,15 @@ public Layout getCurrentLayout() {
                     } catch (InterruptedException ie) {
                         log.warn("Interrupted Exception in layout helper.", ie);
                     }
+                } else if (re instanceof ServerNotReadyException) {
+                    log.warn("Server still not ready. Waiting for server to start "
+                            + "accepting requests.");
```

Comment:
You are right, invalidation should be triggered automatically after a seal with a WrongEpochException.
I'll remove this.